src/dom-to-image-more.js

```diff
@@ -430,10 +436,10 @@
             })
             .then(util.escapeXhtml)
             .then(function (xhtml) {
-                return `<foreignObject x="0" y="0" width="100%" height="100%">${xhtml}</foreignObject>`;
+                return `<foreignObject width="${width}" height="${height}">${xhtml}</foreignObject>`;
             })
             .then(function (foreignObject) {
-                return `<svg xmlns="http://www.w3.org/2000/svg" width="${width}" height="${height}">${foreignObject}</svg>`;
+                return `<svg xmlns="http://www.w3.org/2000/svg">${foreignObject}</svg>`;
```

Comment:
I personally think we should set a viewport here:

```suggestion
                return `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 ${width} ${height}">${foreignObject}</svg>`;
```

Reply:
Or we could do both! https://github.com/whatwg/html/issues/3510#issuecomment-369982529 - seems like SVGs just need both for good cross-browser compat..

Reply:
`viewBox` is problematic [because those values are floats, not integers.](https://github.com/whatwg/html/issues/3510#issuecomment-370101075)

Reply:
This lib would be generating SVGs without a bounding box. I consider this to be "canvas mode" - it's an image but without any notion of natural dimensions. Less useful and likely a major change in semver.
It's more convenient for devs to pass the natural dimension matching the original element. `width="${width}" height="${height}"` is valuable imho

Reply:
I've restore the width and height on the SVG image.